x/emissions/proto/emissions/v5/query.proto

```diff
@@ -0,0 +1,1241 @@
+syntax = "proto3";
```

Comment:
Do we need query.proto v5?

Reply:
Yes, it is because we added new module params(`InitialRegretQuantile`, `PnormSafeDiv`) so need to update query and tx proto